packages/client/css/generic/input.css

```diff
@@ -0,0 +1,66 @@
+.Input {
+  margin-bottom: 16px;
+
+  & label {
+    display: block;
+    color: var(--gray-300);
+
+    font-weight: bold;
+    font-size: 16px;
+
+    text-transform: uppercase;
+    margin-bottom: 4px;
+    margin-left: 13px;
+
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
+       -moz-user-select: none;
+            user-select: none;
+  }
+
+  & input {
+    width: 280px;
+    padding: 13px 16px;
+
+    font-family: inherit;
+    font-size: 18px;
+
+    color: var(--gray-100);
+    background: var(--gray-900);
+
+    border: 0;
+    border-bottom: 1px solid var(--gray-500);
+    &:focus { border-color: var(--blue); }
+
+    &[disabled] {
+      cursor: not-allowed;
+      background: var(--gray-700);
+      opacity: .8;
+    }
+  }
+}
+
+.Textarea {
+  width: 90%;
+  max-width: 600px;
+  padding: 13px 16px;
+
+  min-height: 160px;
+  resize: vertical;
+
+  font-family: inherit;
+  font-size: 18px;
+
+  color: var(--gray-100);
+  background: var(--gray-900);
+
+  border: 0;
+  border-bottom: 1px solid var(--gray-500);
+  &:focus { border-color: var(--blue); }
+
+  &[disabled] {
+    cursor: not-allowed;
+    background: var(--gray-700);
+    opacity: .8;
```

Comment:
There seems to be some inconsistency with whether or not to include a leading 0. We should choose a standard (my vote is for including the 0) and mention it in CONTRIBUTING.md.

Reply:
My vote is 1000% for including the zero.

Reply:
wrong